features/step_definitions/resources.rb

```diff
@@ -123,6 +123,13 @@
   end
 end
 
+Given /^I wait until the status of #{QUOTED} (\w+) becomes :(.+)$/ do |name, type, status|
+  _resource = resource(name, type)
+  ready_timeout = 10 * 60
+  @result = _resource.wait_till_status(status.to_sym, user, ready_timeout)
+end
```

Comment:
nice

Reply:
I like this step and I would like to keep it. The thing is that looking at replication controller it does not have a status of itself, we are looking at the statuses of its pod.
perhaps in your scenario you need to use step `a pod becomes ready with labels:` or `status becomes :([^\s]*?) of( exactly)? ([0-9]+) pods? labeled:`. WDYT?

Reply:
@akostadinov  agree with you. I'll update the feature file if I hit scenario failure again. Then How about the pr, merge it or close it?